src/EventGridExtension/OutputBinding/EventGridOutputAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.Azure.WebJobs.Description;
+
+namespace Microsoft.Azure.WebJobs.Extensions.EventGrid
+{
+    /// <summary></summary>
+    /// <seealso cref="System.Attribute" />
+    [AttributeUsage(validOn: AttributeTargets.Parameter)]
+    [Binding]
+    public sealed class EventGridOutputAttribute : Attribute
```

Comment:
we should rename it to just `EventGridAttribute`, to align with [others](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob#output)